antibody.go

```diff
@@ -28,29 +39,97 @@ func New(home string, r io.Reader) *Antibody {
 
 // Bundle processes all given lines and returns the shell content to execute
 func (a *Antibody) Bundle() (result string, err error) {
-	var g errgroup.Group
-	var lock sync.Mutex
-	var shs []string
-	scanner := bufio.NewScanner(a.r)
-	for scanner.Scan() {
-		l := scanner.Text()
-		g.Go(func() error {
-			l = strings.TrimSpace(l)
-			if l != "" && l[0] != '#' {
-				s, err := bundle.New(a.Home, l).Get()
-				lock.Lock()
-				shs = append(shs, s)
-				lock.Unlock()
-				return err
+	file := a.r
```

Comment:
I think this could be simplified.

Basically, revert to the previous code, and do something like this:

```go
// not a good name...
type Shell struct {
	Position int
	Shs []string
}
```

And the function would be something like:

```go
var g errgroup.Group
var lock sync.Mutex
var shs []Shell
var idx int
scanner := bufio.NewScanner(a.r)
for scanner.Scan() {
	l := scanner.Text()
	idx++
	idx := idx
	g.Go(func() error {
		l = strings.TrimSpace(l)
		if l != "" && l[0] != '#' {
			s, err := bundle.New(a.Home, l).Get()
			lock.Lock()
			shs = append(shs, Shell{idx, s})
			lock.Unlock()
			return err
		}
		return nil
	})
}

// order the result and return
```

What do you think?